src/xcode/ENA/ENA/Source/AppDelegate.swift

```diff
@@ -204,4 +202,26 @@ extension AppDelegate: ENATaskExecutionDelegate {
 		}
 
 	}
+
+	/// This method performs a check for the current exposure detection state. Only if the risk level has changed compared to the
+	/// previous state, a local notification is shown.
+	/// NOTE: This method will always return true.
+	private func executeExposureDetectionRequest(task: BGTask, completion: @escaping ((Bool) -> Void)) {
+
+		let detectionMode = DetectionMode.fromBackgroundStatus()
+		riskProvider.configuration.detectionMode = detectionMode
+
+		riskProvider.requestRisk(userInitiated: false) { risk in
```

Comment:
Hi @ChristianKienle ,
Internally it has a timer to prevent the task from being executed too often, right?

Reply:
It does not have a timer but it checks whether or not enough time has passed already.